nmrfx-processor/src/main/java/org/nmrfx/processor/datasets/vendor/bruker/BrukerData.java

```diff
@@ -203,6 +203,7 @@ public Dataset toDataset(String datasetName) throws IOException {
             dataset.setLabel(i, nucLabel + (i + 1));
             dataset.syncPars(i);
         }
+        dataset.setNFreqDims(dataset.getNDim());
         Integer ncProc = getParInt("NC_proc,1");
```

Comment:
Bruker.toDataset automatically assumes all freqDomains are true, unsure if there is a way to determine from the files whether each dimensions is processed, right now, a pseudo 3D would have all 3 freqDomains set to true